migration/migrate_modifications_in_directory_to_composite_modification.sql

```diff
@@ -0,0 +1,31 @@
+begin transaction isolation level serializable;
+
+with
+
+    dir_element_with_m_id as (select *, gen_random_uuid() as new_id from directory.element where type = 'MODIFICATION'),
```

Comment:
Can't we only work with this common table expression and avoid all the other "as".
Actually, we only need to work with oldId and newId, they don't change